.gitignore

```diff
@@ -216,3 +216,4 @@ aclinit.json
 
 # Deep Learning Experiment
 outputs/
+.out
```

Comment:
`.out` is project specific? maybe more it in your project folder .gitignore

Reply:
sure